services/app-web/src/components/FileUpload/TableWrapper/TableWrapper.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { FileItem, FileItemT } from '../FileProcessor/FileProcessor'
+import { Table } from '@trussworks/react-uswds'
+
+type TableWrapperProps = {
+    fileItems: FileItemT[]
+    deleteItem: (id: FileItemT) => void
+    retryItem: (item: FileItemT) => void
+}
+
+export const TableWrapper = ({
+    fileItems,
+    deleteItem,
+    retryItem,
+}: TableWrapperProps): React.ReactElement => {
+    return (
+        <Table fullWidth>
+            <thead>
+                <tr>
+                    <th>Document name</th>
```

Comment:
Small stylistic thing. Not blocking. Do we want an empty state here?
![Screen Shot 2022-01-27 at 9 41 07 AM](https://user-images.githubusercontent.com/10750442/151394135-77cfe17a-972a-4af3-8cf9-c0d8d6a83764.png)

Reply:
@haworku yes I think we have an empty state design here https://www.figma.com/file/a0PPSr65q1G798zV4bhzzO/State-submission-form?node-id=6432%3A59072